docs/developer/other/IDE_INTEGRATION.md

```diff
@@ -0,0 +1,31 @@
+# IDE Integration
+
```

Comment:
This documentation says what to do. I can follow these steps. But why? What do these steps accomplish? How is it better to do this rather than just open the project via IDE without having first done these steps?